iron-input.html

```diff
@@ -137,16 +136,22 @@
       return canDispatch;
     },
 
-    _dispatchEventFirefoxIE: function() {
+    /**
+     * @this {Node}
+     * @param {!Event} event
+     * @return {boolean}
+     */
+    _dispatchEventFirefoxIE: function(event) {
       // Due to Firefox bug, events fired on disabled form controls can throw
       // errors; furthermore, neither IE nor Firefox will actually dispatch
       // events from disabled form controls; as such, we toggle disable around
       // the dispatch to allow notifying properties to notify
       // See issue #47 for details
       var disabled = this.disabled;
       this.disabled = false;
-      this._origDispatchEvent.apply(this, arguments);
+      var defaultPrevented = this._origDispatchEvent.call(this, event);
```

Comment:
isn't this equivalent to `var defaultPrevented = this._origDispatchEvent(event);`?